src/Windows/Containers/Test/OutputMock.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.R.Common.Core.Output;
+
+namespace Microsoft.R.Containers.Windows.Test {
+    public class OutputMock : IOutput {
```

Comment:
There is `Microsoft.Common.Core.Test.Fakes.Shell.TestOutputService` that writes output in log file that becomes test artifact.